packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.tsx

```diff
@@ -781,15 +703,23 @@ export class ContextualMenu extends BaseComponent<IContextualMenuProps, IContext
     this._scrollIdleTimeoutId = this._async.setTimeout(() => { this._isScrollIdle = true; }, this._navigationIdleDelay);
   }
 
-  private _onItemMouseEnter(item: any, ev: React.MouseEvent<HTMLElement>) {
+  private _onItemMouseEnter = (item: any, ev: React.MouseEvent<HTMLElement>): void => {
+    this._onItemMouseEnterBase(item, ev, ev.currentTarget as HTMLElement);
+  }
+
+  private _onItemMouseEnterBase = (item: any, ev: React.MouseEvent<HTMLElement>, target: HTMLElement): void => {
```

Comment:
Maybe let target be optional since it's optional when going into focusMouseEvent. Not sure about this though, I'll leave it up to you